frontend/src/metabase/lib/expressions/process.js

```diff
@@ -1,10 +1,10 @@
-import { parse } from "metabase/lib/expressions/parser";
-import { compile } from "metabase/lib/expressions/compile";
-import { suggest } from "metabase/lib/expressions/suggest";
-import { syntax } from "metabase/lib/expressions/syntax";
-
 // combine compile/suggest/syntax so we only need to parse once
 export function processSource(options) {
+  const parse = require("./parser").parse;
+  const compile = require("./compile").compile;
+  const suggest = require("./suggest").suggest;
+  const syntax = require("./syntax").syntax;
```

Comment:
I opt for going with the simple/primitive `require` style, aiming for the most minimal diff and the least riskiest change.

An alternative will be to use dynamic import (since `import` normally has to be top-level), but that mandates some config changes in Babel and/or webpack, which potentially opens a can of worms and which isn't desperately need to achieve this lazy loading anyway.